pom.xml

```diff
@@ -41,7 +41,13 @@
         <groupId>org.apache.maven.plugins</groupId>
         <version>3.12.1</version>
       </plugin>
-    </plugins>
+	<plugin>
```

Comment:
I think we can remove this. We resolved your liquibase issues by bringing up the project out of the eclipse workspace into separate intellij projects